src/frontend/Player/createPlyrPlayer.ts

```diff
@@ -149,5 +149,13 @@ export const createPlyrPlayer = (
   });
   /**************** End dispatch time updated *********************/
 
+  window.addEventListener('unload', () => {
+    if (false === isInitialized) {
```

Comment:
```suggestion
    if (isInitialized) {
```

Is there a reason to specifically look for `false`? As far as I understand `isInitialized` is always a boolean

Reply:
Yes this one of the calisthenics rules. I agree here I could test if `isInitialized` is true and then call the `terminated` staement. It's an habit for me now to do this comparison.

Reply:
IMO you both need to agree on this to make the code the homogeneous possible. Once you agreed, this should be enforced by new linter rules.